setup.md

```diff
@@ -21,6 +21,25 @@ To install and run Flutter, your development environment must meet these minimum
 * **Tools**: Flutter depends on these command-line tools being available on your environment.
   * `bash`, `mkdir`, `rm`, `git`, `curl`, `unzip`
 
+## Get the Flutter SDK
+
+To get Flutter, use `git` to clone the repository and then add the `flutter` tool to your path:
+
+<pre>
+$ git clone https://github.com/flutter/flutter.git -b alpha
+$ export PATH=`pwd`/flutter/bin:$PATH
+</pre>
+
+To complete the setup and to validate that you’ve set everything up correctly, run the following
+command:
```

Comment:
I've revised the instructions to read as follows:
~~~
Run the following command to see if there are any dependencies 
you need to complete the setup:

$ flutter doctor

The first time your run the flutter command, it will download its 
dependencies and compile itself. Subsequent runs should be much faster.

Once you have installed any missing dependencies, run the flutter doctor 
command again to verify that you’ve set everything up correctly.